This patch was diffed by me, thanks to ferdy, jokey
and some others for their IRC-Help.


--- Acrobat7.0/bin/acroread	2005-05-18 09:05:35.000000000 +0200
+++ Acrobat7.0/bin/acroread	2005-05-18 09:06:29.000000000 +0200
@@ -1,6 +1,8 @@
 #!/bin/sh
 #
 
+FC=~/.adobe/Acrobat/7.0/Cache/UnixFnt07.lst
+[ -s ${FC} ] && rm ${FC} && touch ${FC}
 
 setLangDEU()
 {
